### PR TITLE
Removes the old Job/Flow API

### LINKED ISSUE #65: Remove non execution API from Sqoop
The jobs/flow approach is superseded by execution and the latest version of Thermometer no longer supports testing for it.

We need to remove or rewrite the old API in terms of Execution.

### PR DESCRIPTION
**Pull request by stephanh**
This PR removes the old Job/Flow API in preparation for the Scala 2.11 upgrade where Thermometer no longer jobs Scalding Jobs/Flows.

* Adds tests for the existing console jobs
* Changes the console jobs to use execution API as backend
* Removes the old Job/Flow based API.

This closes #65.

### CODE REVIEW

File: src/test/scala/au/com/cba/omnia/parlour/ExportSqoopSpec.scala [hunk 1]
```diff
@@ -74,122 +65,64 @@ class ExportSqoopSpec  extends ThermometerSpec with ExportDb { def is = s2"""
...
-      withEnvironment(path(resourceUrl.toString)) {
-        val expected = newData ++ oldData
-        val table = tableSetup(oldData)
-        val dsl = createDsl(table)
-
-        val source = Csv(exportDir, "|").createTap(Read)
-        val sink   = TableTap(dsl.toSqoopOptions)
-        val flow   = new ExportSqoopFlow("endToEndFlow", dsl, Some(source), Some(sink))
+    val table = tableSetup(oldData)
+    val dsl = createDsl(table)
```
Comment (shmookey) [inline src/test/scala/au/com/cba/omnia/parlour/ExportSqoopSpec.scala]: Maybe line these up (also in a couple instances later on)

Comment (stephanh) [inline src/test/scala/au/com/cba/omnia/parlour/ExportSqoopSpec.scala]: Done.

**Comment by kfish** [comment]
nice cleanup!

**Comment by stephanh** [comment]
Any :+1: for this PR?

**Comment by shmookey** [comment]
:+1:

**Pull request merged**